### PR TITLE
Add helpers that makes dealing with `Bytes` values easier

### PR DESCRIPTION
**Pull request by lutter**
This makes the functionality enabled by [this PR](https://github.com/graphprotocol/graph-node/pull/3271) more accessible

### CODE REVIEW

**Comment by lutter** [comment]
Fixed code formatting

File: common/value.ts [hunk 1]
```diff
@@ -129,6 +140,42 @@ export class Value {
...
+        let arr = this.toArray()
+        return '[' + arr.map<string>((elt) => elt.displayData()).join(', ') + ']'
+      case ValueKind.NULL:
+        return 'null'
+      case ValueKind.BYTES:
+        return this.toBytes().toHexString()
+      case ValueKind.BIGINT:
+        return this.toBigInt().toString()
+      default:
+        return `Unknwon data (kind = ${this.kind})`
```
Comment (otaviopace) [inline common/value.ts:175]: Typo in `unknown`.

File: common/value.ts [hunk 2]
```diff
@@ -16,6 +16,17 @@ export enum ValueKind {
...
+const VALUE_KIND_NAMES = [
+  'String',
+  'Int',
+  'BigDecimal',
+  'Bool',
+  'Array',
+  'Null',
+  'Bytes',
+  'BigInt',
+]
```
Comment (otaviopace) [inline common/value.ts:28]: Shouldn't `Bool` and `Null` be lowercase?

File: common/collections.ts [hunk 3]
```diff
@@ -67,7 +67,9 @@ export class ByteArray extends Uint8Array {
   }
 
   /**
-   * Input length must be even.
+   * Convert the string `hex` which must consist of an even number of
```
Comment (otaviopace) [inline common/collections.ts:70]: Thanks!!

File: common/collections.ts [hunk 4]
```diff
@@ -268,13 +285,34 @@ export class Bytes extends ByteArray {
     return changetype<Bytes>(uint8Array)
   }
 
+  /**
+   * Convert the string `hex` which must consist of an even number of
+   * hexadecimal digits to a `ByteArray`. The string `hex` can optionally
+   * start with '0x'
+   */
+  static fromHexString(str: string): Bytes {
```
Comment (otaviopace) [inline common/collections.ts:293]: Thanks pt2! lol

It's kind of annoying we have to make duplicate methods for those so users don't need to do the upcasting.

**Review by otaviopace — APPROVED** [review]

**Comment by otaviopace** [comment]
I approved by mistake 😅 

But yeah I read it all, it seems mostly correct 👏

File: common/value.ts (hunk 2 above)
Comment (lutter) [reply common/value.ts:28]: Wasn't sure how that's shown elsewhere. Just changed it.

File: common/value.ts (hunk 1 above)
Comment (lutter) [reply common/value.ts:175]: Fixed

**Review by otaviopace — APPROVED** [review]

**Pull request merged**